samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/delivery/DeliveryBean.java

```diff
@@ -1284,7 +1284,7 @@ public String validatePassword() {
     if (StringUtils.isBlank(password)) {
     	return "passwordAccessError";
     }
-    if(StringUtils.isNotBlank(getSettings().getPassword()) && !StringUtils.equals(StringUtils.trim(password), StringUtils.trim(getSettings().getPassword()))) {
+    if(StringUtils.isNotBlank(getSettings().getPassword()) && !StringUtils.equals(StringUtils.trim(password), ComponentManager.get(FormattedText.class).convertFormattedTextToPlaintext(StringUtils.trim(getSettings().getPassword())))) {
```

Comment:
There is another use of FormattedText in this file it would be ideal to assign it once.

Reply:
change made! thank you for your comment.